devito/dse/rewriters.py

```diff
@@ -406,6 +411,32 @@ def _extract_sum_of_products(self, cluster, template, **kwargs):
 
         return cluster.rebuild(processed)
 
+    @dse_pass
+    def _skewing(self, cluster, template, **kwargs):
+        """
+        Skew the accesses along the time Dimension.
+        """
+
+        skew_factor = self.DEFAULT_SKEW_FACTOR
+
+        _, sub_iterators, directions = cluster.ispace.args
+
+        skew_dim, mapper, intervals = None, {}, []
+
+        for i in cluster.ispace.intervals:
+            if i.dim.is_Time:
+                intervals.append(Interval(i.dim, 0, 0))
+                skew_dim = i.dim
+            else:
+                mapper[i.dim] = i.dim + skew_factor*skew_dim
+                intervals.append(Interval(i.dim, -skew_factor*skew_dim,
+                                          -skew_factor*skew_dim))
+
+        processed = xreplace_indices(cluster.exprs, mapper)
+        ispace = IterationSpace(intervals, sub_iterators, directions)
+        cluster = Cluster(processed, ispace, cluster.dspace, cluster.guards)
```

Comment:
as I said, this is supposed to be ``guards=cluster.guards ``

Reply:
I had changed already, thanks, forgot to push :-(.
Had it commited locally.